golden_liquid.json

```diff
@@ -104,7 +113,8 @@
       "name": "identifiers, trailing question mark assign",
       "template": "{% assign foo? = 'hello' %}{{ foo? }}",
       "tags": [
-        "strict"
+        "strict",
```

Comment:
Is it possible to put the tags in alphabetical order, or other deterministic order? I'm concerned that regenerating this might make changes to the file that are not semantically different.